### DISCUSSION TITLE
Does Wurst Client work?

### DISCUSSION THREAD

**Discussion by MOST0AFA** [Sinytra/Connector#968] [Q&A] [answered]
I keep trying to use Wurst but it just crashes,any fixes?

**Comment by Su5eD** [comment, marked as answer]
Custom server APIs (such as Mohist, Magma, Arclight) are not supported. This also applies to launchers for mobile platforms (Pojav, FoldCraft) and pirated launchers.

Such platforms provide unpredictable environments in which mods might not function as expected, and therefore we can't provide support for them. Ensuring mod compatibility is the platform's duty, just like Connector tries its best to achieve parity when running Fabric mods on Forge.

This is explained in our [FAQ](https://github.com/Sinytra/Connector/discussions/689).

**Comment by Aripander08-832** [comment]
This person isnt asking about server arcitecture, They are asking about a utility mod for fabric
